docs/site-replication/README.md

```diff
@@ -19,7 +19,7 @@ This feature is built on top of multi-site bucket replication feature. It enable
 1. Initially, only **one** of the sites being added for replication may have data. After site-replication is successfully configured, this data is replicated to the other (initially empty) sites. Subsequently, objects may be written to any of the sites, and they will be replicated to all other sites.
 2. Only the **LDAP IDP** is currently supported.
 3. At present, all sites are **required** to have the same root credentials.
-4. At present it is not possible to **add a new site** to an existing set of replicated sites or to **remove a site** from a set of replicated sites.
+4. At present it is not possible to **remove a site** from a set of replicated sites.
```

Comment:
We shouldn't talk about this as a limitation - this is totally expected that they can't remove a site. 

```suggestion
4.  **removing a site** is not allowed from a set of replicated sites once configured. 
```